dev-workflow-ui/webContent/includes/components/casesTable.xhtml

```diff
@@ -22,7 +22,7 @@
 
     <p:column filterBy="#{case.state}" styleClass="icon-row" sortBy="#{case.state}">
       <h:outputText id="caseState" styleClass="si si-lg si-#{casesIvyDevWfBean.getStateIcon(case.state)}" />
-      <p:tooltip for="caseState" value="State: #{case.state}" position="top" />
+      <p:tooltip for="caseState" value="State: #{case.businessState} (#{case.state})" position="top" />
```

Comment:
use here case.state and case.technicalState as well

Reply:
@alexsuter the problem is that this is `ICase` so it has only `businessState` and `state`
I could change the `TaskModel` to be the same as this tho?

Reply:
CaseModel could be anyway a good idea. But yes please change TaskModel, that this is the same.